content/js/index.md

```diff
@@ -66,17 +68,14 @@ available globally (`window.skygear` or just `skygear`).
 <script src="//code.skygear.io/js/skygear/latest/skygear.min.js"></script>
 ```
 
-<a name="intro-portal"></a>
-## Basics of Skygear JS SDK
-
 <a name="container"></a>
-### Container
+### Set up Container
```

Comment:
- The "Include the SDK in your project" includes both installing and including the SDK in the website.
- "Setting up your JS app" refers to configuring the api key and endpoint